### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by TimothyMothra**
### Changes
- add 460 and 461 to versions list
- add instructions for using preview versions of .NET with Visual Studio

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ To successfully build the sources on your machine, make sure you've installed th
     - .NET 6
     - .NET 7
 
+Note: .NET has an annual release cycle and we include the preview version in our test matrix.
```
Comment (cijothomas) [inline .github/CONTRIBUTING.md:31]: if you are here, can you also update the pre-requisites above? we need to have all .NET Frameworks . net452,nrt46,net461,net471 .. we dont list all of them now!

Comment (TimothyMothra) [reply .github/CONTRIBUTING.md:31]: net452 is already included! :)

I'm discussing with Raj if we can remove net46 and net461 from this repo.

I can't find any references to net471.

Comment (cijothomas) [reply .github/CONTRIBUTING.md:33]: image
These 4 I had to install to build the solution locally. Yes there is no net471

Comment (TimothyMothra) [reply .github/CONTRIBUTING.md:33]: Done

**Pull request merged**